sdk/core/azure-common/azure/common/client_factory.py

```diff
@@ -50,7 +51,18 @@ def _client_resource(client_class, cloud):
 def get_client_from_cli_profile(client_class, **kwargs):
     """Return a SDK client initialized with current CLI credentials, CLI default subscription and CLI default cloud.
 
-    This method will fill automatically the following client parameters:
+    *Disclaimer*: This is NOT the recommended approach to authenticate with CLI login, this method is deprecated.
+    use https://pypi.org/project/azure-identity/ and AzureCliCredential instead. See example code below:
+
+    .. code:: python
+
+        from azure.identity import AzureCliCredential
+        from azure.mgmt.compute import ComputeManagementClient
+        client = ComputeManagementClient(AzureCliCredential(), subscription_id)
+
+    This method is not working for CLI installation after 3/2021 (version 2.21.0 of azure-cli-core).
```

Comment:
This method will not work for CLI installations after 3/2021

Reply:
changed the complete sentence after a comment from Johan on ambiguous guidance between dates and version